api/src/services/users/get_saved_searches.py

```diff
@@ -22,7 +22,9 @@ def get_saved_searches(
 
     search_params = SavedSearchListParams.model_validate(raw_search_params)
 
-    stmt = select(UserSavedSearch).where(UserSavedSearch.user_id == user_id)
+    stmt = select(UserSavedSearch).where(
+        (UserSavedSearch.user_id == user_id) & (UserSavedSearch.is_deleted.isnot(True))
+    )
```

Comment:
I'm not sure this syntax is accepted - at the very least it's not documented: https://docs.sqlalchemy.org/en/20/tutorial/data_select.html#the-where-clause

where also accepts multiple comma separated clauses, so can just trim it to:
```suggestion
    stmt = select(UserSavedSearch).where(UserSavedSearch.user_id == user_id, UserSavedSearch.is_deleted.isnot(True))
```